gnosis/safe-cmm-app-react
# Feature/#42/deploy page component

# Description

Part of #42 

Basic deploy page layout done
![screenshot_2020-09-11_16-36-10](https://user-images.githubusercontent.com/43217/92981008-f8f94900-f44c-11ea-979d-61c9c7d814b8.png)

Design https://projects.invisionapp.com/d/main/default/#/console/20213446/427462295/preview

# To Test
1. Install dependencies `npm i`
1. Start storybook `npm run storybook`
1. Visit http://localhost:6006/?path=/story/pages-deploy--default

- [ ] Style should match design

## Not in this PR
- Any logic
- Any validation
- ~Any action (for example, selecting a token will have no effect)~ now there are some minimal actions

# Background

First part of #42, will work on wiring up the logic next.

Went with an approach of splitting component into smaller fragments and passing down props with a context provider to avoid repetition.
So far, so good. Will see if I have any issues with this approach.
